apis/Google.Datastore.V1Beta3/Google.Datastore.V1Beta3.Snippets/DatastoreDbSnippets.cs

```diff
@@ -90,10 +91,11 @@ public void StructuredQuery()
             {
                 Filter = Filter.Equal("author", "Jane Austen")
             };
-            DatastoreQueryResults results = db.RunQuery(query);
-            // DatastoreQueryResults implements IEnumerable<Entity>, but you can
-            // call AsEntityResults(), AsBatches() or AsResponses() to see the query
-            // results in whatever way makes most sense for your application.
+            LazyDatastoreQuery results = db.RunQuery(query);
```

Comment:
Okay. We'd have `RunQueryAsync` as well, returning a `Task<DatastoreQueryResults>`. I'll try to get a new version of this PR done tomorrow - it'll probably just be an extra commit on top of this one.